internal/bminventory/inventory.go

```diff
@@ -2987,6 +2987,31 @@ func (b *bareMetalInventory) processDiskSpeedCheckResponse(ctx context.Context,
 	return b.hostApi.SetDiskSpeed(ctx, h, diskPerfCheckResponse.Path, diskPerfCheckResponse.IoSyncDuration, exitCode, nil)
 }
 
+func (b *bareMetalInventory) updateDomainNameResolutionResponse(ctx context.Context, host *models.Host, domainResolutionResponseReporter string) error {
+	var (
+		err                      error
+		domainResolutionResponse models.DomainResolutionResponse
```

Comment:
Please declare parameters where they are used, not at the beginning of the function

Reply:
done